crates/aranya-daemon/src/aranya.rs

```diff
@@ -41,14 +44,21 @@ pub enum SyncResponse {
 pub struct Client<EN, SP, CE> {
     /// Thread-safe Aranya client reference.
     aranya: Arc<Mutex<ClientState<EN, SP>>>,
+    /// Thread-safe Peer Cache reference.
+    /// Lock must be aquired after [`Self::aranya`]
+    peer_caches: Arc<Mutex<BTreeMap<Addr, PeerCache>>>,
```

Comment:
Since this is the only cache for this object, the field name could be more concise:
```suggestion
    cache: Arc<Mutex<BTreeMap<Addr, PeerCache>>>,
```

Reply:
Shortened it to "caches" since there are multiple. We're keeping [one per peer](https://github.com/aranya-project/aranya/pull/212#discussion_r2050987534)